src/components/Band.js

```diff
@@ -1,19 +1,14 @@
-import React, { Component } from 'react';
+import React from 'react';
 
-class Band extends Component {
+const Band = ({ deleteBand, id, name }) => {
+  const handleOnClick = id => deleteBand(id);
 
-  handleOnClick = () => {
-    this.props.deleteBand(this.props.id)
-  }
-
-  render() {
-    return(
-      <div>
-        <li>{this.props.name}</li>
-        <button onClick={this.handleOnClick}>Delete</button>
-      </div>
-    );
-  }
+  return (
+    <li className="band">
```

Comment:
`Bands` establishes a `ul`, and if the child thereof is not an `li` it is semantically invalid. Let's wrap these children in an `li` here (and lean on that css to do our styling work.)